Jenkinsfile

```diff
@@ -386,6 +386,40 @@ rocm_head:
     */
   }
 },
+rocm_clang:
+{
+  node('hip-rocm')
+  {
+    String hcc_ver = 'rocm-head'
+    String from_image = 'ci_test_nodes/rocm-clang/ubuntu-16.04:latest'
```

Comment:
Are you building hip-clang from scratch?

Reply:
New docker image with all libraries required to test HIP-HCC with HIP-clang compiler.